### PR TITLE
docs: add note for disabling pytest plugin

### LINKED ISSUE #3671: pytest plugin starts tracer when plugin is disabled
### Which version of dd-trace-py are you using?

1.1.1

### Which version of pip are you using?

```
# pip --version
pip 22.0.4 from /usr/local/lib/python3.8/site-packages/pip (python 3.8)
```

### Which version of the libraries are you using?

```
# pip freeze
attrs==21.4.0
ddsketch==2.0.1
ddtrace==1.1.1
iniconfig==1.1.1
packaging==21.3
pluggy==1.0.0
protobuf==3.20.1
py==1.11.0
pyparsing==3.0.8
pytest==7.1.2
six==1.16.0
tenacity==8.0.1
tomli==2.0.1
```

### How can we reproduce your problem?

Check out https://gist.github.com/majorgreys/f799970f731fdf70d865d18325e1c34d and run:

```
docker run -it --rm (docker build -q .)
```

### What is the result that you get?

`ddtrace` modules are imported by the Python interpreter. Importing any of these modules starts the global tracer.

The workaround for this is to disable the `ddtrace` plugin with `-p no:ddtrace` in the command.

### What is the result that you expected?

`ddtrace` should not be the modules imported by the Python interpreter.

### PR DESCRIPTION
**Pull request by majorgreys**
Fixes #3671

Preview

Screen Shot 2022-05-04 at 3 55 51 PM

### CODE REVIEW

File: releasenotes/notes/pytest-disable-note-dababc18228d1f85.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+---
+fixes:
+  - |
+    pytest: Add note for disabling ddtrace plugin as workaround for side-effects
```
Comment (Kyle-Verhoog) [inline releasenotes/notes/pytest-disable-note-dababc18228d1f85.yaml:4]:
```suggestion
    pytest: Add note for disabling ddtrace plugin as workaround for side-effects

```

File: ddtrace/contrib/pytest/__init__.py [hunk 2]
```diff
@@ -18,6 +18,22 @@
...
+   package and starting a global tracer.
+
+   If this is causing issues for your pytest runs where traced execution of
+   tests is not enabled, you can deactivate the plugin::
+
+     [pytest]
+     addopts = -p no:ddtrace
+
+   See the `pytest documentation
+   <https://docs.pytest.org/en/latest/how-to/plugins.html#deactivating-unregistering-a-plugin-by-name>`_
```
Comment (Kyle-Verhoog) [inline ddtrace/contrib/pytest/__init__.py:33]: maybe want to link to a specific version so the reference doesn't break if pytest moves their docs around

**Review by Kyle-Verhoog — APPROVED** [review]
2 nits otherwise lgtm!

File: ddtrace/contrib/pytest/__init__.py (hunk 2 above)
Comment (majorgreys) [inline ddtrace/contrib/pytest/__init__.py:33]:
```suggestion
   <https://docs.pytest.org/en/7.1.x/how-to/plugins.html#deactivating-unregistering-a-plugin-by-name>`_
```

**Review by Kyle-Verhoog — APPROVED** [review]

**Review by P403n1x87 — APPROVED** [review]

**Pull request merged**